From ae6b15b2a03f250a0d37d9ba4cd452fa2543be99 Mon Sep 17 00:00:00 2001
From: Arian <arian.kulmer@web.de>
Date: Thu, 28 Nov 2024 20:05:39 +0530
Subject: [PATCH 06/12] Use standard gallery intent

*Change com.android.camera.action.REVIEW to android.provider.action.REVIEW
---
 smali/com/android/camera/GalleryHelper.smali | 4 ++--
 smali/com/android/camera/Util.smali          | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/smali/com/android/camera/GalleryHelper.smali b/smali/com/android/camera/GalleryHelper.smali
index 588deb4da..b7b61e7d6 100644
--- a/smali/com/android/camera/GalleryHelper.smali
+++ b/smali/com/android/camera/GalleryHelper.smali
@@ -713,7 +713,7 @@
 
     move-result p0
 
-    const-string v0, "com.android.camera.action.REVIEW"
+    const-string v0, "android.provider.action.REVIEW"
 
     if-eqz p0, :cond_0
 
@@ -782,7 +782,7 @@
 
     const-string p1, "GalleryHelper"
 
-    const-string v1, "gotoGallery: com.android.camera.action.REVIEW"
+    const-string v1, "gotoGallery: android.provider.action.REVIEW"
 
     .line 10
     invoke-static {p1, v1, p0}, Lcom/android/camera/log/Log;->d(Ljava/lang/String;Ljava/lang/String;[Ljava/lang/Object;)V
diff --git a/smali/com/android/camera/Util.smali b/smali/com/android/camera/Util.smali
index 5ddc84ed4..a3ef69781 100644
--- a/smali/com/android/camera/Util.smali
+++ b/smali/com/android/camera/Util.smali
@@ -220,7 +220,7 @@
 
 .field public static final REQUEST_CODE_OPEN_MIUI_EXTRA_PHOTO:I = 0x8c35
 
-.field public static final REVIEW_ACTION:Ljava/lang/String; = "com.android.camera.action.REVIEW"
+.field public static final REVIEW_ACTION:Ljava/lang/String; = "android.provider.action.REVIEW"
 
 .field public static final REVIEW_ACTION_IN_MULTI_WINDOW:Ljava/lang/String; = "com.android.camera.action.SPILIT_SCREEN_REVIEW"
 
-- 
2.49.0

